keps/sig-node/2053-downward-api-hugepages/kep.yaml

```diff
@@ -0,0 +1,48 @@
+title: Downward API HugePages
+kep-number: 2053
+authors:
+  - "@derekwaynecarr"
+owning-sig: sig-node
+participating-sigs: []
+status: implementable
+creation-date: 2020-06-18
+reviewers:
+  - "@dashpole"
+  - "@sjenning"
+approvers:
+  - "@dashpole"
+  - "@sjenning"
+  - "@dchen1107"
+prr-approvers:
+  - "deads2k"
+  - "johnbelamaric"
+  - "wojtek-t"
+see-also:
+  - "/keps/sig-node/20190129-hugepages.md"
+replaces: []
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.20"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v1.20"
+  beta: "v1.21"
+  stable: "v1.22"
+
+# The following PRR answers are required at alpha release
+# List the feature gate name and the components for which it must be enabled
+feature-gates:
+  - name: DownwardAPIHugePages
+    components:
+      - kubelet
```

Comment:
clarify what the feature gate controls in the kubelet... I don't think we have a great mechanism for enabling a gate in one component independently, so the "enable in the kubelet 2 releases before the apiserver" bit might be tricky to do with a single gate. Options are multiple gates, or leaving the kubelet code path ungated to be triggered by the presence of the data in the API

Reply:
I prefer kubelet behavior is ungated, apiserver is gated, will clarify.